src/main/java/frc/robot/commands/BasicDrive/TurnLeft.java

```diff
@@ -39,8 +40,8 @@ public void initialize() {
     SmartDashboard.putNumber("getYaw", currentYaw);
     desiredYaw = currentYaw - degrees;
     // used to account for sign change of yaw after it reaches -180*
-    if (desiredYaw < -180) {
```

Comment:
This constant isn't needed because it is a part of a math equation where it is certain the value won't change